app/constants/task-type.js

```diff
@@ -3,5 +3,7 @@ module.exports = {
   CREATE_WORKLOAD: 'CREATE-WORKLOAD',
   PROCESS_IMPORT: 'PROCESS-IMPORT',
   PROCESS_REDUCTIONS: 'PROCESS-REDUCTIONS',
-  PROCESS_ADJUSTMENTS: 'PROCESS-ADJUSTMENTS'
```

Comment:
Need to add a row for `CREATE_COURT_REPORTS ` in `get-worker-for-task`

Reply:
Added